template/{{.input_root_dir}}/README.md.tmpl

```diff
@@ -124,53 +127,60 @@ contained in the following files:
-| Data Scientist                | Get started writing ML code for a brand new project                          | {{ if (eq .input_include_feature_store `yes`) }}[ML quickstart](docs/ml-developer-guide-fs.md).{{ else }}[ML quickstart](docs/ml-developer-guide.md){{ end }} |
+| Data Scientist                | Get started writing ML code for a brand new project                          | {{ if (eq .input_setup_cicd_and_project `CICD_and_Project`)}}[project README](./{{template `project_name_alphanumeric_underscore` .}}/README.md) {{ else }} README when project is initialized {{ end }}|
 | MLOps / DevOps                | Set up CI/CD for the current ML project   | [MLOps setup guide](docs/mlops-setup.md)                                                                                                                            |
 | Data Scientist                | Update production ML code (e.g. model training logic) for an existing project | [ML pull request guide](docs/ml-pull-request.md)                                                                                                                    |
-| Data Scientist                | Modify production model ML assets, e.g. model training or inference jobs  | [ML asset config guide]({{template `project_name_alphanumeric_underscore` .}}/assets/README.md)                                                     |
-
-## Monorepo
-
-It's possible to use the repo as a monorepo that contains multiple projects. All projects share the same workspaces and service principals.
-
-For example, assuming there's existing repo with root directory name `monorepo_root_dir` and project name `project1`
-1. Create another project from `databricks bundle init` with project name `project2` and root directory name `project2`.
-2. Copy the internal directory `project2/project2` to root directory of existing repo `monorepo_root_dir/project2`.
-{{ if or (eq .input_cicd_platform `github_actions`) (eq .input_cicd_platform `github_actions_for_github_enterprise_servers`) -}}
-3. Copy yaml files from `project2/.github/workflows/` to `monorepo_root_dir/.github/workflows/` and make sure there's no name conflicts.
+| Data Scientist                | Modify production model ML assets, e.g. model training or inference jobs  | {{ if (eq .input_setup_cicd_and_project `CICD_and_Project`)}}[ML asset config guide]({{template `project_name_alphanumeric_underscore` .}}/assets/README.md) {{ else }} ML assets README when project is initialized {{ end }} |
+
+## Setting up CI/CD
+This stack comes with a workflow to set up CI/CD for projects that can be found in
+{{ if (eq .input_cicd_platform `azure_devops`)}}
+`.azure/devops-pipelines/deploy-cicd.yml`.
+{{ else }}
+`.github/workflows/deploy-cicd.yml`.
```

Comment:
Nit: the `.github/workflows/deploy-cicd.yml` or `.azure/devops-pipelines/deploy-cicd.yml` is on a different line
<img width="1303" alt="Screenshot 2024-02-02 at 2 18 44 AM" src="https://github.com/databricks/mlops-stacks/assets/12734110/b5288e06-510e-4b6b-907e-02806272fc97">